tests/testthat/test-addin.R

```diff
@@ -0,0 +1,43 @@
+context("use_addin")
+
+test_that("use_addin() creates the first addins.dcf as promised, default", {
+  scoped_temporary_package()
+  use_addin()
+
+  addin_dcf <- readLines(proj_path("inst", "rstudio", "addins.dcf"))
+  expect_true(
```

Comment:
Could you make these tests a little more minimal? They're currently fragile because if we change any of the lines in `addins.dcf` we'll need to change all the tests.